test/QuadForm/ShortVectors.jl

```diff
@@ -45,7 +45,7 @@
                      2448, 4320, 2982, 2402, 2364, 512, 256, 306 ]
 
   # Floating point bounds
-  L = integer_lattice(gram = QQ[1//2 0; 0 1//3])
```

Comment:
I would duplicate the tests instead of removing the old tests.

Reply:
Good point, I will modify this.